lib/src/services/jira.ts

```diff
@@ -184,7 +246,7 @@ export const BoardSummary = T.type({
 
 export type BoardSummary = T.TypeOf<typeof BoardSummary>;
 
-export type EnhancedIssue = Issue & {
+export type EnhancedIssue = JiraIssue & {
```

Comment:
Let's revert this rename as well. So the type should be `Issue`.

Reply:
Changed all instances of `JiraIssue` back to `Issue`. To keep consistent syntax, the same was done for `CloudJiraIssue` and `OnPremJiraIssue`